code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -1310,24 +1313,12 @@
 	else
 		update_icon_dropped()
 
-// Note: For effects on subtypes, use the emp_effect() proc instead
+// Note: Does NOT return EMP protection value from parent call or pass it on to subtypes
 /obj/item/bodypart/emp_act(severity)
 	var/protection = ..()
```

Comment:
More stuff accidentally reverted

Reply:
lovely

Reply:
septic fixed this, resolving

Reply:
septic did not fix this, but i did eventually, phew